packages/writing-style/styles/commercetools/Acronyms.yml

```diff
@@ -55,6 +55,7 @@ exceptions:
   - RAML
   - REST
   - RFC # "Request For Comment" would confuse readers even more.
+  - RMF # "Rest Modeling Framework"
```

Comment:
Please remove this, the writing style is not for our own internal content but for user facing content.  In that context, "RMF" must be explained before use. 

@davifantasia  the root cause is that we are still validating release notes etc. with the writing style which does not make sense. 

Can be excluded with a special comment for now (or you remove "md" extensions from writing style checking right here)